### PR TITLE
OSDOCS-7287: adds 4.12.28 to RNs and fixes typo

### PR DESCRIPTION
**Pull request by JoeAldinger**
Version(s):
4.12

Issue:
https://issues.redhat.com/browse/OSDOCS-7287

Link to docs preview:
https://63256--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes#ocp-4-12-28

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-12-release-notes.adoc
Comment (skopacz1) [inline release_notes/ocp-4-12-release-notes.adoc:3930]: Everything looks good, but these two advisory links are giving me 404 errors. If it's because they won't be publicly available until the 9th, disregard this comment.

**Review by skopacz1 — COMMENTED** [review]
Everything looks good, just left one comment over something that's likely not an issue. I'm still new to reviewing, so please wait until @adellape approves of my review before proceeding. Thanks!

**Review by adellape — APPROVED** [review]
@skopacz1 @JoeAldinger Peer review comments LGTM 👍

**Pull request closed without merging**
